Sources/Purchasing/PurchasesDelegate.swift

```diff
@@ -47,15 +47,32 @@ import Foundation
     /**
      * Called when a user initiates a promotional in-app purchase from the App Store.
      * If your app is able to handle a purchase at the current time, run the deferment block in this method.
-     * If the app is not in a state to make a purchase: cache the defermentBlock,
-     * then call the defermentBlock when the app is ready to make the promotional purchase.
-     * If the purchase should never be made, you don't need to ever call the defermentBlock and
-     * ``Purchases`` will not proceed with promotional purchases.
-
+     * If the app is not in a state to make a purchase: cache the `startPurchase` block,
+     * then call the `startPurchase` block when the app is ready to make the promotional purchase.
+     *
+     * If the purchase should never be made, you don't need to ever call the block and
+     * ``Purchases`` will not proceed with the promotional purchase.
+     *
+     * This can be tested by opening a link like:
+     * itms-services://?action=purchaseIntent&bundleId=<BUNDLE_ID>&productIdentifier=<SKPRODUCT_ID>
+     *
      * - Parameter product: `StoreProduct` the product that was selected from the app store
+     *
+     * ### Related Articles:
+     * - [Apple Documentation](https://rev.cat/testing-promoted-in-app-purchases)
      */
+    @objc optional func purchases(_ purchases: Purchases,
+                                  readyForPromotedProduct product: StoreProduct,
```

Comment:
extra picky here, feel free to ignore...but maybe `readyToPurchasePromotedProduct`?

Reply:
Then the whole method would become `purchases(_: readyToPurchasePromotedProduct:purchase:)` instead of "ready for promoted product purchase".